nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/components/explorer/grid-list/registry/nf-registry-grid-list-viewer.html

```diff
@@ -121,6 +121,38 @@
                                     </td-step>
                                 </td-steps>
                             </div>
+                            <div *ngIf="droplet.bundleType" id="nifi-registry-explorer-grid-list-viewer-droplet-container-details-change-log">
+                                <td-steps mode="vertical">
+                                    <td-step label="Version {{snapshotMeta.version}} ({{nfRegistryService.getHumanReadableSize(snapshotMeta.contentSize)}}) - {{snapshotMeta.timestamp | amTimeAgo}}" sublabel="by {{snapshotMeta.author}}"
+                                             *ngFor="let snapshotMeta of droplet.snapshotMetadata; let i = index"
+                                             [active]="i === 0 ? true : false">
+                                        <div fxLayout="column" fxLayoutAlign="space-between stretch">
+                                            <div fxLayout="row" class="mat-body-3">
+                                                <p>
+                                                Build Information<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>NiFi API version</b>: {{snapshotMeta.systemApiVersion}}<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Build branch</b>: {{snapshotMeta.buildInfo.buildBranch}}<br />
```

Comment:
Recommend capitalizing the second word in these lines:
```suggestion
                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Build Branch</b>: {{snapshotMeta.buildInfo.buildBranch}}<br />
```